pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -48,21 +50,47 @@ pandas_df.dtypes
 
 Viewing the DataFrame in Jupyter looks like this. 
 
-```python, evaluate=False
+```python show_pandas, evaluate=False
 pandas_df
 ```
 
 @@include[df-samples-output.md](static/df-samples-output.md)
 
-## PNG Overviews
-
-It is oftentimes convenient to have a simple overview image of the data, particularly if it spans a very wide area.
 
 ## GeoTIFFs
 
-GeoTIFF is one of the most common file formats for spatial data. 
+GeoTIFF is one of the most common file formats for spatial data, providing flexibility in data encoding, representation, and storage. RasterFrames provides a specialized Spark DataFrame writer for rendering a RasterFrame to a GeoTiff.
+
+One downside to GeoTIFF is that it is not a big data native format. To create a GeoTIFF all the data to be encoded has to be in the memory of one compute node (in Spark parlance, this is a "collect"), limiting it's maximum size substantially compared to that of a full cluster environment. When rendering GeoTIFFs in RasterFrames, you either need to specify the dimensions of the output raster, or be aware of how big the collected data will end up being.
+
+Fortunately, we can use the cluster computing capability to downlample the data into a more manageble saze. For sake of example, let's render a simple RGB overview image of our scene as a small raster:
+
+```python write_geotiff
+import os.path
+from docs import docs_dir
+cat = """  
+red,green,blue
+{},{},{}
+""".format(scene('01'), scene('04'), scene('03'))
+outfile = os.path.join(docs_dir(), 'geotiff-overview.tif')
+rf = spark.read.raster(catalog=cat, catalog_col_names=['red', 'green', 'blue'])
+rf.write.geotiff(outfile, crs='EPSG:4326', raster_dimensions=(256, 256))
+```
+
+Wiew it with `rasterio` to check the results:
+
+```python view_geotiff
+import rasterio
+import numpy as np
+from rasterio.plot import show, show_hist
+ 
+with rasterio.open(outfile) as src:
+    cells = np.clip(src.read(), 0, 1800).astype('float32')
+    show(cells)
+    show_hist(src, bins=50, lw=0.0, stacked=False, alpha=0.3,
+        histtype='stepfilled', title="Overview Histogram")
+```
```

Comment:
I would also like to see an example where we select a contiguous subset and write it out at full resolution and in native CRS. 

Just for completeness. 

I think that would be a quite common use case also. Consider for example writing out ML classification model predictions for an area intersecting a polygon.

Reply:
https://github.com/locationtech/rasterframes/issues/228